.checkstyle/suppressions.xml

```diff
@@ -5,5 +5,8 @@
         "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-  <!-- No suppressions here -->
+  <suppress checks="CyclomaticComplexity" files="YammerMetricsCollector.java"/>
+  <suppress checks="NPathComplexity" files="YammerMetricsCollector.java"/>
+  <suppress checks="JavaNCSS" files="YammerMetricsCollector.java"/>
+  <suppress checks="ClassFanOutComplexity" files="YammerMetricsCollector.java"/>
```

Comment:
Should you use annotations to suppress these instead of suppressing it here? That is what we normally do -> it is better coupled with the code and easier to see and maintain.

Reply:
Right, if it's what Strimzi usually does, yes let's use annotations. Done